app.js

```diff
@@ -1,80 +1,142 @@
+//1.
 let englishGreeting = "Hello, World!"
 console.log(englishGreeting);
 let kurdishGreeting = "Silav, Cihan!"
 console.log(kurdishGreeting);
 let dutchGreeting = "Hoi, Wereld!"
 console.log(dutchGreeting);
 
+//2.
 let me = "I'm awesome!"
 console.log(me);
 
+//3.1
 let x;
-console.log("I think the value of x is a number");
-console.log("The value of x is 6");
+
+//3.2
+console.log("I think the value of x will be: undefined");
+
+//3.3
+console.log(x);
+
+//3.4
 let x = 6;
-console.log("I think the value of x is 6");
+
+//3.5
+console.log("the value of x is: 6");
+
+//3.6
 console.log(x);
 
+//4.1
 let y ="Sivan";
+
+//4.2
 console.log("I think the value of y is a name");
+
+//4.3
 console.log(y);
+
+//4.4
 const y = "Hassam";
-console.log("There will be an error because now I have
-             the same variable with different values.");
+
+//4.5
+console.log("There will be an error because now I have the same variable with different values.");
+
+//4.6
 console.log(y);
 
+//5.
 const z = 7.25;
+
+//5.1
 console.log(z);
+
+//5.2
 const a = z;
+
+//5.3
 console.log(a);
+
+//5.4
 var round =Math.round(7.25);
+
+//5.6
 console.log(round)
 
-let arrays =
-console.log("I think variable arrays is related to animals");
+//6.1
+let arrays;
+
+//6.2
+console.log("I think the value will be undefined");
+
+//6.3
 console.log(arrays);
+
+//6.4
 let myFavoriteAnimals = [cat, dog, sheep];
+
+//6.5
 console.log(myFavoriteAnimals);
+
+//6.6
 let favoriteAnimalOfDaan = "baby pig";
-const animals = "myFavoriteAnimals" + "" + "favoriteAnimalOfDaan";
+const animals = myFavoriteAnimals + "" + favoriteAnimalOfDaan;

```

Comment:
Ah this one is still incorrect,you should use `myFavoriteAnimals.push('baby pig')`. 
Read more about it here: https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
Changed to: const animals = myFavoriteAnimals.push('baby pig');
Just last week, we have covered the .push() method.